spec/features/idv/in_person_spec.rb

```diff
@@ -494,4 +495,29 @@
       expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
     end
   end
+
+  context 'when the USPS enrollment fails during enter password' do
+    before do
+      user = user_with_2fa
+      sign_in_and_2fa_user(user)
+      begin_in_person_proofing(user)
+      complete_prepare_step(user)
+      complete_location_step
+      complete_state_id_step(user, first_name: 'usps client error')
```

Comment:
Idle musing: it's easy to miss what this is doing, and instead just skim this and see it as doing normal steps, in which case it's unclear why an error would be expected. We had experimented a bit with descriptively-named helper methods to make some of this clearer, though I don't know where that ended up. I wonder if it would make sense to put this in a method like `complete_up_to_enter_password_step_with_usps_client_error` and just call that in the `before` here? (It's totally possible this is a bad idea. Just mentioning it in case it resonates.)

Reply:
Yeah I feel like it is unclear as well... I think this is because I was working around the fact that we are using the mock proofer in the tests. Maybe I should change the tests to use the real proofer, and mock the http endpoint. That would make the test setup have an explicit step for mocking the `OptInIPPApplicant` endpoint to return a 400 response.

Reply:
Actually this is turning out to be a bigger fish than I want to fry right now... I ended up adding a comment above that line that states why the first_name is set to `usps client error`. I think it would be nice to improve these feature tests at a later date.

Reply:
> I wonder if it would make sense to put this in a method like complete_up_to_enter_password_step_with_usps_client_error and just call that in the before here?

I get wanting to make the test easier to follow.  In the spirit of write everything twice, I wanted to point out that making a helper method which isn't used much could be a premature optimization.